src/components/List/List.jsx

```diff
@@ -1,25 +1,26 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
 import { getPosts } from '../../selectors/Selectors';
+import Post from '../Post/Post';
 
-import Post from '../../components/Post/Post';
+const List = () => {
 
-export default function List() {
-  const posts = useSelector(getPosts);
+  const post = useSelector(getPosts);
```

Comment:
you aren't grabbing a single post but an array so this should probably be posts.